### PR TITLE
add Huawei Ascend NPU setup guide

### PR DESCRIPTION
**Pull request by xuedinge233**
This PR adds the setup instructions for Huawei Ascend NPU. Please refer to the remainder of the guide for instructions on other devices.

### CODE REVIEW

**Comment by xuedinge233** [comment]
@microsoft-github-policy-service agree [company="Huawei"]

**Comment by xuedinge233** [comment]
@microsoft-github-policy-service agree company="Huawei"

**Comment by hipudding** [comment]
This PR is a sub task of #5827

File: docs/_tutorials/accelerator-setup-guide.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ tags: getting-started
 - [Introduction](#introduction)
 - [Intel Architecture (IA) CPU](#intel-architecture-ia-cpu)
 - [Intel XPU](#intel-xpu)
+- [NPU](#npu)
```
Comment (loadams) [inline docs/_tutorials/accelerator-setup-guide.md:11]: Do you want to add Huawei here?

Comment (hipudding) [reply docs/_tutorials/accelerator-setup-guide.md:11]: I think it's better to change NPU to Huawei Ascend NPU here. @xuedinge233

Comment (xuedinge233) [reply docs/_tutorials/accelerator-setup-guide.md]: Ok, I have finished the modification

**Review by loadams — APPROVED** [review]

**Pull request merged**